docs/source/standard.md

```diff
@@ -255,39 +255,41 @@ The ability of the organisation to support users who want to browse the data ava
+| TRE operators should have procedures in place for rapid incident response                                                                   | By simulating potential security breaches or incidents during security testing, organisations running TREs can assess their incident response procedures and ensure a swift and effective response when a real incident occurs.                                                                                                                                                                                                                  |
+| Regular security testing can increase stakeholder trust in TREs                                                                             | Knowledge that regular security testing occurs will help to ensure stakeholders, including researchers and data providers, can trust that the data they work with or are responsible for is secure within a TRE.                                                                                                                                                                                                                                  |
 
-### 4.2 Security testing
+### 4.2 Encryption
 
-The ability of the organisation to gain assurance in the security of an IT system by testing or attempting to breach some or all of that system's security.
+The ability of the organisation to deploy and manage encryption to protect information assets, including data for TRE research projects.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement                                             | Guidance                                                                                                                                                                                                                                                                                                                                                                                                          |
+| ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TREs can encrypt data as an extra layer of protection | Encryption can be seen as an extra layer of protection for sensitive data already protected by the security controls of a TRE. This prevents unauthorised access to the data even if the storage media is compromised. Approved researchers can be granted access to the mechanism to decrpyt the data before using it in their research.                                                                         |
```

Comment:
```suggestion
| TREs must encrypt project and user data at rest | Encryption can be seen as an extra layer of protection for sensitive data already protected by the security controls of a TRE. This prevents unauthorised access to the data even if the storage media is compromised. Approved researchers can be granted access to the mechanism to decrpyt the data before using it in their research.                                                                         |
```

This feels like an essential and (relatively) easy target.
For on prem there are filesystems supporting encryption, and tools like `dm-crypt` and `LUKS`.
For public cloud this is something you get for free (almost certainly).
I'm not familiar with private cloud but I would imagine it is well supported to encrypt either at the physical storage or virtual storage level.

Some things I think we need to define where,

What data is being encrypted? I would propose

- 'project' data, _i.e._ the data brought in for work which is very likely personal and sensitive
- 'user' data, _i.e._ the working files of a project which are likely to hold copies, slices of the 'project' data or otherwise reveal sensitive data (_e.g._ through hard coded row/column names).

Encrypted again who?

- I'm imagining the encryption is protecting again a bad actor who is not a TRE user or admin. _E.g._ someone steals a harddrive, someone breaches your cloud providers data centre, a bad actor employed by your cloud provider.
- A further step would be to encrypt so that TRE admins are not able to see the data.